app/controllers/boards_controller.rb

```diff
@@ -31,7 +31,10 @@ def update
     board = current_user.owned_boards.find(params[:id])
     board.update_attributes(board_params)
 
-    redirect_to board
+    respond_to do |format|
+      format.html { redirect_to board }
```

Comment:
@as-cii just a quick question here: since bip, is there a case where a `format.html` is ever hit?